Arelle/Arelle
# Conformance suite configuration & download script

#### Reason for change
Resolves #395 
For people interested in running standard conformance suites against Arelle locally, a bit of legwork is needed to actually get all of the needed files in the right place. This PR adds the ability to automatically download public conformance suite files from the source (and direct to a membership page for those that are not public).

#### Description of change
As part of this, I implemented a conformance suite configuration system so we don't need to duplicate CS details across a test script, a runner script, and a download script.

#### Steps to Test
See results on fork [here](https://github.com/aaroncameron-wk/arelle-public/pull/80).
XBRL 2.1 and IXBRL runs will fail because the corresponding .zip files needs to be swapped out on S3 since they do not match the ZIP from the xbrl.org source. Once approved, QA or I can swap out the zips and re-run the failed tests before merging.

**review**:
@Arelle/arelle
